src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -64,6 +64,14 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.KeyVault.2.3.0-preview\lib\net452\Microsoft.Azure.Management.KeyVault.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=15.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@sergey-shandar please remove the references added in this file; they should be already handled elsewhere by common code

Reply:
We need to merge this PR https://github.com/Azure/azure-powershell/pull/5075 to remove Azure SDK references.